entries/slideToggle.xml

```diff
@@ -50,7 +50,7 @@ $( "#clickme" ).click(function() {
       <img class="column three" src="/resources/0042_06_30.png" alt=""/>
       <img class="column three" src="/resources/0042_06_31.png" alt=""/>
       <img class="column three" src="/resources/0042_06_32.png" alt=""/>
-      <figcaption>figure 2</figcaption>
+      <figcaption>Figure 2 - Illustration of the <code>slideToggle()</code> effect</figcaption>
```

Comment:
@kswedberg the :+1: goes to which comment? (my entry or Aurelio's suggestion?)
Thanks :-)